RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -247,6 +228,17 @@ - (BOOL)mapView:(RCTMapboxGL *)mapView annotationCanShowCallout:(id <MGLAnnotati
     return YES;
 }
 
+- (void)selectAnnotationAnimated:(NSUInteger)annotationInArray
+{
+    if (_annotations) {
+        if ([_annotations count] <= annotationInArray) return NSLog(@"%s", "Annotation not found in annotations array");
```

Comment:
If the caller passes in a bogus ID, that’s programmer error, so assert (with `NSAssert`) instead of logging.